src/stactools/core/create.py

```diff
@@ -38,14 +39,14 @@ def item(href: str, read_href_modifier: Optional[ReadHrefModifier] = None) -> It
         proj_bbox = dataset.bounds
         proj_transform = list(dataset.transform)[0:6]
         proj_shape = dataset.shape
-    proj_geometry = shapely.geometry.mapping(shapely.geometry.box(*proj_bbox))
-    geometry = stactools.core.projection.reproject_geom(
-        crs, "EPSG:4326", proj_geometry, precision=6
+    geom = stactools.core.projection.reproject_shape(
+        crs, "EPSG:4326", shapely.geometry.box(*proj_bbox), precision=6
     )
-    bbox = list(shapely.geometry.shape(geometry).bounds)
+    bbox = list(geom.bounds)
+    geojson = json.loads(json.dumps(shapely.geometry.mapping(geom)))
```

Comment:
I added it because `mapping` was outputting tuples which was not matching test expectations. But tuples dump as arrays, so maybe the test is too restrictive?

Reply:
I think we should relax/rework the test. We should always support putting a `shapely.geometry.mapping` into `pystac.Item.geometry`, and so if that starts failing, we should fix it in PySTAC. Removing `loads->dumps` here adds another "ecosystem test" to that desired code path.